src/site/_includes/components/BaseCard.js

```diff
@@ -56,7 +57,7 @@ class BaseCard {
   }
 
   renderThumbnail(url, img, alt) {
-    const imagePath = path.join(url, img);
+    const imagePath = img.match(/^http(s?):\/\//) ? img : path.join(url, img);
```

Comment:
This seems like the BaseCard knows too much about how it's being used. It would probably make more sense to override this method in a subclass and add this. But see my comments above about treating the newsletters like any other piece of content. We may not need to do this if we copy their feature image into the same directory as the newsletter's html.